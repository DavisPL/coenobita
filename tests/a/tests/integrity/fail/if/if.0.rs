#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    // Since the value assigned to `x` is implicitly affected by
    // crate `c`, this example should fail to type check
    #[cnbt::integrity({bin}{bin})]
    let x = if c::boolean() {
        5
    } else {
        6
    };
}