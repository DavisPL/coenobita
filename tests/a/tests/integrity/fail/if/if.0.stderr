error: expected {root}{root}, found {root}{c,root}
 --> tests/integrity/fail/if/if.0.rs:9:9
  |
9 |         5
  |         ^
